assets/dev/js/frontend/elements-handler.js

```diff
@@ -1,21 +1,32 @@
+import accordionHandler from './handlers/accordion';
```

Comment:
Look for any anonymous function throughout the handlers files and convert it to arrow functions